Misc/NEWS.d/next/Library/2024-08-16-19-13-21.gh-issue-123067.Nx9O4R.rst

```diff
@@ -0,0 +1 @@
+Fix quadratic complexity in parsing cookies with backslashes.
```

Comment:
Maybe mention the module being affected?

Reply:
I did it initially, but the reference to `http.cookies` does not work.

Reply:
Do you mean that

```rst
Fix quadratic complexity in parsing cookies with backslashes by :mod:`http.cookies`.
```

does not work?

Reply:
It seems that I made a typo in the module name.